src/main/java/nextstep/domain/Line.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Line {
+
+    private static final RandomTransverseBarStrategy DEFAULT_TRANSVERSE_BAR_STRATEGY = new RandomTransverseBarStrategy();
+
+    private List<Boolean> points;
+
+    public Line(int countOfPerson) {
+        this(drawTransverseBar(countOfPerson, DEFAULT_TRANSVERSE_BAR_STRATEGY));
+    }
+
+    public Line(List<Boolean> points) {
+        if (!validateNotOverlap(points)) {
+            throw new IllegalArgumentException("횡단선이 겹치면 안됩니다.");
+        }
+        this.points = points;
+    }
+
+    public static boolean validateHasOneOrMoreTransverseBar(List<Boolean> line) {
+        return line.stream()
+                .anyMatch(Boolean::booleanValue);
+    }
+
+    public static boolean validateNotOverlap(List<Boolean> line) {
+        return IntStream.range(0, line.size() - 1)
+                .mapToObj(i -> new LineValidateOverlapDto(line.get(i), line.get(i + 1)).isOverlap())
+                .noneMatch(Boolean::booleanValue);
+    }
+
+    public List<Boolean> getPoints() {
+        return points;
+    }
```

Comment:
Collections.unmodifiablelist()를 활용하여 read-only로 리턴한다면 외부에서 컬렉션의 변조를 막을 수 있습니다.
```suggestion
    public List<Boolean> getPoints() {
        return Collections.unmodifiablelist(points);
    }
```